src/js/sidebar/form-document-settings/index.js

```diff
@@ -196,6 +198,19 @@ class FormDocumentSettings extends Component {
 										} )
 									}
 								/>
+								{ 'checkout' === location && 'yes' === showTitle &&
+								<TextControl
+									label={ __( 'Free Access Plan Form Title', 'lifterlms' ) }
+									value={ freeApTitle }
+									onChange={ ( value ) =>
+										setFormMetas( { _llms_form_title_free_access_plans: value } )
+									}
+									help={ __(
+										"The form title to be shown for free access plans. Leave it empty if you don't want to show any title for free acces plan checkout forms.",
```

Comment:
```suggestion
										"The form title to be shown for free access plans.",
```

Reply:
I had removed that piece... I guess I didn't save it before committing. sorry.